defender/connection.py

```diff
@@ -18,6 +19,8 @@ def get_redis_connection():
     """ Get the redis connection if not using mock """
     if config.MOCK_REDIS:  # pragma: no cover
         return MOCKED_REDIS  # pragma: no cover
+    elif config.DEFENDER_REDIS_NAME:  # pragma: no cover
+        return caches[config.DEFENDER_REDIS_NAME].get_master_client()
```

Comment:
we talked about this in the issue #81 instead of adding the and to the elif, maybe we do a check above, and then display an error message and fail if the DEFENDER_REDIS_NAME is set but not available in caches. This way we still fail right away, and we give it a nice error message to let them know what is going on vs just a generic KeyError.